src/qt/locale/bitcoin_th.ts

```diff
@@ -1,315 +1,2642 @@
+        <translation type="unfinished">(แต่กระเป๋าเงินนี้ไม่สามารถลงนามธุรกรรมได้)</translation>
+    </message>
+    <message>
+        <source>(But this wallet does not have the right keys.)</source>
+        <translation type="unfinished">(แต่กระเป๋าเงินนี้ไม่มีคีย์ที่ถูกต้อง)</translation>
+    </message>
+    <message>
+        <source>Transaction is fully signed and ready for broadcast.</source>
+        <translation type="unfinished">การทำธุรกรรมลงนามครบถ้วนแล้วและพร้อมสำหรับการประกาศ</translation>
+    </message>
+    <message>
+        <source>Transaction status is unknown.</source>
+        <translation type="unfinished">สถานะของธุรกรรมไม่ทราบแน่ชัด</translation>
+    </message>
+</context>
+<context>
+    <name>PaymentServer</name>
+    <message>
+        <source>Payment request error</source>
+        <translation type="unfinished">ข้อผิดพลาดในการขอชำระเงิน</translation>
+    </message>
+    <message>
+        <source>Cannot start bitcoin: click-to-pay handler</source>
+        <translation type="unfinished">ไม่สามารถเริ่มต้นบิตคอยน์: ตัวจัดการคลิกเพื่อจ่าย</translation>
+    </message>
+    <message>
+        <source>URI handling</source>
+        <translation type="unfinished">การจัดการ URI</translation>
+    </message>
+    <message>
+        <source>'bitcoin://' is not a valid URI. Use 'bitcoin:' instead.</source>
+        <translation type="unfinished">'bitcoin://' ไม่ใช่ URI ที่ถูกต้อง ใช้ 'bitcoin:' แทน</translation>
+    </message>
+    <message>
+        <source>Cannot process payment request because BIP70 is not supported.
+Due to widespread security flaws in BIP70 it's strongly recommended that any merchant instructions to switch wallets be ignored.
+If you are receiving this error you should request the merchant provide a BIP21 compatible URI.</source>
+        <translation type="unfinished">ไม่สามารถดำเนินการคำขอชำระเงินได้เนื่องจาก BIP70 ไม่ได้รับการรองรับ
+เนื่องจากมีข้อบกพร่องด้านความปลอดภัยอย่างแพร่หลายในการใช้ BIP70 จึงขอแนะนำอย่างยิ่งให้เพิกเฉยต่อคำแนะนำของพ่อค้าในการเปลี่ยนกระเป๋าเงิน
+หากคุณได้รับข้อผิดพลาดนี้คุณควรขอให้พ่อค้าจัดเตรียม URI ที่รองรับ BIP21</translation>
+    </message>
+    <message>
+        <source>URI cannot be parsed! This can be caused by an invalid Bitcoin address or malformed URI parameters.</source>
+        <translation type="unfinished">ไม่สามารถแยก URI ได้! สิ่งนี้อาจเกิดจากที่อยู่ Bitcoin ที่ไม่ถูกต้องหรือพารามิเตอร์ URI ที่ผิดรูปแบบ</translation>
+    </message>
+    <message>
+        <source>Payment request file handling</source>
+        <translation type="unfinished">การจัดการไฟล์คำขอการชำระเงิน</translation>
+    </message>
+</context>
+<context>
+    <name>PeerTableModel</name>
+    <message>
+        <source>User Agent</source>
+        <extracomment>Title of Peers Table column which contains the peer's User Agent string.</extracomment>
+        <translation type="unfinished">ตัวแทนผู้ใช้
+(Pronounced: Dtua-Thaen Phu-Chai)</translation>
```

Comment:
I noticed a couple of translations have these pronunciations, is that expected according to our translation guidelines (do we have any)?

Reply:
No, it is not expected. Removed on Transifex.